src/state/calculationOptions.ts

```diff
@@ -5,6 +5,7 @@ export const calculationOptionsState = atom<CalculatorTypes.CalculationOptions>(
   key: 'calculationOptions',
   default: {
     calculateHTML: false,
-    calculateSDEs: false
+    calculateSDEs: false,
+    reportType: 'individual'
```

Comment:
Added to prevent uncontrolled -> controlled component warnings from react. Is there a more appropriate default value for reportType?

Reply:
I think `individual` makes the most sense personally.